### PR TITLE
webtesting-iv-challenge

### PR DESCRIPTION
**Pull request by tsarg88**

### CODE REVIEW

File: helpers/testModel.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    return 0
+  }
+}
+
+function getAll() {
+  return db('test');
+}
+
+function findById(id) {
+  return db('test').where('id',id);
```
Comment (mikaelacurrier) [inline helpers/testModel.js:38]: if both the values are the same, you can say `where({id})`

File: helpers/testModel.spec.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const request = require('supertest');
+
+const db = require('../data/dbConfig.js');
+const dbModel = require('./testModel.js');
+
+describe('hobbits model', () => {
+    describe('insert()', () => {
+        afterEach(async () => {
+            await db('test').truncate()
+        })
```
Comment (mikaelacurrier) [inline helpers/testModel.spec.js:10]: this is a cool method - just make sure you never use this in production - don't want to accidentally delete all your data when you run tests. This is why the cleanup method is nice

**Review by mikaelacurrier — COMMENTED** [review]
Tigran, looks great! You seem to be all prepared for the sprint!

File: helpers/testModel.js (hunk 1 above)
Comment (tsarg88) [reply helpers/testModel.js:38]: OK, thanks! Will correct it.

**Pull request merged**